packages/toast/test/standalone-toast.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from "react"
+import { render } from "@chakra-ui/test-utils"
+import { createStandaloneToast } from "../src"
+import theme from "@chakra-ui/theme"
+
+test("Standalone toast renders correctly", async () => {
+  const toast = createStandaloneToast(theme)
+  const title = "Yay!"
+  const description = "Something awesome happened"
+
+  const ToastContainer = () => {
+    useEffect(() => {
+      toast({
+        title,
+        description,
+        duration: 4000,
+        isClosable: true,
+      })
+    }, [])
+
+    return null
+  }
+
+  const { findAllByText } = render(<ToastContainer />)
+
+  const allByTitle = await findAllByText(title)
```

Comment:
Can you switch this to use `screen.findAllByText` rather than destructuring from `render`?

Reply:
✅  done